scripts/mkcloud

```diff
@@ -1033,6 +1045,11 @@ Optional
         set new cloudsource for upgrade process
     TESTHEAD='' | 1  (default='')
         use Media from Devel:Cloud:Staging and add test update repositories (except for GM* targets)
+    raidvolumenumber (default=1)
+        The number of disks to join into the software RAID for each node.
+        Default value of 1 means just one system disk and no RAID.
+    want_raidtype (default='raid1')
+        The type of RAID to create.
```

Comment:
want_raidtype doc could also be moved into qa_crowbarsetup's onadmin_help function. and squash it into the 3rd commit